fastccd_support_ioc/fastccd_support_ioc.py

```diff
@@ -244,6 +246,53 @@ async def Initialize(self, instance, async_lib):
         ErrorStatus = pvproperty(dtype=str, value="", read_only=True)
         AutoStart = pvproperty(dtype=bool, value='Off')  # value='On')
 
+        BiasState = pvproperty(dtype=ChannelType.ENUM,
+                           enum_strings=["Unknown", "On", "Powering On...", "Powering Off...", "Off", ],
+                           value="Unknown")
+
+        @BiasState.startup
+        async def BiasState(self, instance, async_lib):
+            global com_lock
+            self.async_lib = async_lib
+            com_lock = async_lib.library.locks.Lock()
+
+        @BiasState.putter
+        async def BiasState(self, instance, value):
+            if value != instance.value:
+                print(f"setting state: {value}")
+
+                if value == "Powering On...":
+                    value = await self._power_on_bias(None, None)
+
+                elif value == "Powering Off...":
+                    value = await self._power_off_bias(None, None)
+
+            return value
+
+        async def _power_on_bias(self, instance, value):
+            async with com_lock:
+                power_check_no_bias_clocks()
+                setClocksBiasOn()
+                await self.async_lib.library.sleep(1)
```

Comment:
This wait needs to be greater than 3 sec to assure that the values read back in the check are updated.

Reply:
where does that > 3 second requirement from from? something to do with powering on the bias or something related to caproto/EPICS?